### DISCUSSION TITLE
Vercel API / Integrations - Shape for Body Parameters

### DISCUSSION THREAD

**Discussion by zaiste** [vercel/community#536] [Help]
Hi,

I'm building a Vercel integration and using a few of the Vercel REST API endpoints. 

Often times, the possible input data (i.e. the body parameters) are marked as `array` or `object`, but without specifying the actual shape of these parameters (cf. the attached image below)

(`POST` `/v8/projects`)
<img width="790" alt="CleanShot 2022-04-02 at 17 44 07@2x" src="https://user-images.githubusercontent.com/200613/161406408-96e89064-af1d-4da4-9951-baaed68677bf.png">

Would it be possible to provide those types similar to how you describe `Response`s? 

Right now, I manually match the shape for the `body` based on the `Response` interfaces, but sometimes the field names don't match and the process is slightly laborious. It'd be great to have it documented in the Vercel docs directly.

**Comment by samcx** [comment]
Certainly — let me confirm with the team how these should look like!
